lib/orderbook/OrderBook.ts

```diff
@@ -745,15 +742,9 @@ class OrderBook extends EventEmitter {
     pairIds.forEach(async (pairId) => {
       const [baseCurrency, quoteCurrency] = pairId.split('/');
       if (peer.verifiedCurrencies.has(baseCurrency) && peer.verifiedCurrencies.has(quoteCurrency)) {
-        peer.activePairs.add(pairId);
-        verifiedPairs.push(pairId);
+        await peer.activatePair(pairId);
```

Comment:
Minor nit: we could do a `pairsToActivate.push(pairId)` -> `await Promise.all(pairsToActivate)` here for slightly better performance in exchange for slightly worse readability. Up to you - I don't mind the way it currently is.

Reply:
I changed it to use `Promise.all` - not a big deal either way I agree but I like sticking to a pattern of awaiting promises in parallel when they don't depend on each other.